cime_config/testdefs/testlist_clm.xml

```diff
@@ -1390,7 +1389,6 @@
   <test name="SMS_Ld1" grid="f19_g17" compset="I2000Clm50BgcCruGs" testmods="clm/default">
     <machines>
       <machine name="cheyenne" compiler="intel" category="prebeta"/>
-      <machine name="edison" compiler="intel" category="prebeta"/>
```

Comment:
Here and in a couple of other places: you removed an edison prebeta test. I'm thinking you just meant to update the fates test list, not our prebeta test list?

Reply:
@billsacks I purposely did that assuming it would be fine since edison is retired.  Is it better practice not to change since it's across categories/branches?

Reply:
Our CESM testing uses the edison tests for testing on cori (confusingly).

Reply:
Ha, thanks for the clarification.  I'll make the changes.